frontend/javascripts/admin/task/task_create_bulk_view.tsx

```diff
@@ -314,17 +315,19 @@ function TaskCreateBulkView() {
               ]}
             >
               <TextArea
-                className="input-monospace"
                 placeholder="dataset, datasetId, taskTypeId, experienceDomain, minExperience, x, y, z, rotX, rotY, rotZ, instances, minX, minY, minZ, width, height, depth, project[, scriptId, baseAnnotationId]"
                 autoSize={{
                   minRows: 6,
                 }}
-                style={{
-                  fontFamily: 'Monaco, Consolas, "Lucida Console", "Courier New", monospace',
+                styles={{
+                  textarea: {
+                    fontFamily:
+                      'SFMono-Regular, Consolas, "Liberation Mono", Menlo, Courier, monospace',
```

Comment:
🤔 I thought we use Nunito by default. Maybe there is also a monospaced front for this? Maybe we should use this here?  I think @normanrz has a better overview about what font we should use here